src/connection.rs

```diff
@@ -129,25 +128,22 @@ impl WMIConnection {
     fn create_services(&mut self) -> Result<(), Error> {
         debug!("Calling ConnectServer");
 
-
         let mut p_svc = ptr::null_mut::<IWbemServices>();
 
         let object_path = "ROOT\\CIMV2";
         let mut object_path_bstr = WideCString::from_str(object_path)?;
 
         unsafe {
-            check_hres(
-                (*self.loc()).ConnectServer(
-                    object_path_bstr.as_ptr() as *mut _,
-                    ptr::null_mut(),
-                    ptr::null_mut(),
-                    ptr::null_mut(),
-                    0,
-                    ptr::null_mut(),
-                    ptr::null_mut(),
-                    &mut p_svc,
-                )
-            )?;
+            check_hres((*self.loc()).ConnectServer(
```

Comment:
use consistent rustfmt configuration (maybe add a git-hook or something).
this is polluting the PR :(